ignite/metrics/gan/fid.py

```diff
@@ -0,0 +1,218 @@
+import warnings
+from distutils.version import LooseVersion
+from typing import Callable, Optional, Sequence, Union
+
+import numpy as np
```

Comment:
There is docs failure due to this import. I'm not sure if numpy is a hard requirement for torch for cpu release...

Reply:
We need it for calculating trace. Also, it was present in previous commits as well where deploy was successful.

Reply:
The failure started when I changed function return type from `Union[torch.tensor, float]` to just `float`